buildingmotif/ingresses/xlsx.py

```diff
@@ -18,15 +18,18 @@ class XLSXIngress(RecordIngressHandler):
     field of each Record gives the name of the sheet.
     """
 
-    def __init__(self, filename: Path):
+    def __init__(self, filename: Path, limit: Optional[int] = -1):
```

Comment:
Only critique is that this type could be `os.PathLike` instead of `Path`

Reply:
Fixed!